stan/math/opencl/kernels/helpers.hpp

```diff
@@ -9,9 +9,9 @@ namespace math {
 namespace opencl_kernels {
 
 /*
- * Defines some helper macros for the kernels
+ * Defines helper macros for common matrix indexing operations
  */
-static const char* helpers =
+static const char* indexing_helpers =
```

Comment:
I like that you're leaving them all in the same file. It's nice to think about the use-case of someone new to the code trying to get familiar and write their own kernels or edit existing ones. 👍